test/testdata/rewriter/prop_foreign.rb.ast.exp

```diff
@@ -0,0 +1,163 @@
+begin
+  <emptyTree>
+  class <emptyTree><<C <root>>> < (::<todo sym>)
+    def self.<static-init><<static-init>$CENSORED>(<blk>)
```

Comment:
Should this file be `.rewrite-tree.exp`?

Reply:
(done)